.github/workflows/prannouncer.yml

```diff
@@ -0,0 +1,14 @@
+name: "Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 6 * * MON-FRI"
+
+jobs:
+  prannouncer:
```

Comment:
If you wanted to follow guardian naming, you could call it Pronouncer, or even more in line prnouncer.

Reply:
Love it.